libraries/windows_sdk_feature.rb

```diff
@@ -0,0 +1,124 @@
+require_relative "./windows_sdk_helpers"
+class WindowsSdkCookbook
```

Comment:
How do I configure installation location? I assume it's going to go to programfiles, which may not be the right place.